_includes/plugins/powerpaste_import_types.md

```diff
@@ -0,0 +1,7 @@
+The supported string-based values are:
+
+* `clean` - Preserve the structure of the content such as headings, tables, and lists but remove inline styles and classes. This results in simple content that uses the site's CSS stylesheet while retaining the semantic structure from the original document.
```

Comment:
could we add a link to a new separate section that talks about after cleaning, you can add custom classes onto specific elements to further match the sites style?  I think this is very valuable point that TinyMCE can do
eg:
After cleaning, TinyMCE can add additional CSS classes to better match your websites styles. [more details/info]

Reply:
Hmm, might be useful but I feel like it's outside the scope of what the docs usually cover? It's more of a knowledge base / blog post thing I think...